### DISCUSSION TITLE
Iraf installation issue in Mac chip M1 - make sysgen 2>&1 | tee build.log

### DISCUSSION THREAD

**Discussion by estebansilvavilla** [iraf-community/iraf#303] [Q&A] [answered]
Dear all, after some quick checking on the discussion, I think that my problema has not been reported. This si why I open a new discussion.

I followed the instructions on the webpage and everything went smooth until the command:

make sysgen 2>&1 | tee build.log

I attached the result. It seems a problem at some line that is looking for a command that I am not quite sure what it does.

The error message is:

```
.
.
.
----------------------- F2C ---------------------------
+ cd src
+ sh -x mkpkg.sh
+ set -e
+ make -f makefile.u
cc -c -O -w  -g -Wall -O2  -I/Users/esteban/iraf-2.17/include -DNOVOS sysdep.c
sysdep.c:112:3: error: call to undeclared function 'rmdir'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
                rmdir(s);
                ^
sysdep.c:112:3: note: did you mean 'rmtdir'?
sysdep.c:107:1: note: 'rmtdir' declared here
rmtdir(Void)
^
1 error generated.
make[1]: *** [sysdep.o] Error 1
make: *** [sysgen] Error 2
```
I went on but of course nothing worked out. 

Does anyone have gone through this problem before? Any ideas?

Thanks in advance.
[installation_prompt.txt](https://github.com/iraf-community/iraf/files/11511770/installation_prompt.txt)
[installation_prompt.txt](https://github.com/iraf-community/iraf/files/11511773/installation_prompt.txt)

**Comment by estebansilvavilla** [comment, marked as answer]
Found the solution at #298

**Comment by tonysohn** [comment]
I was having the same issue, and was able to solve this problem by adding

`#include <unistd.h>`

at the very top of the `unix/f2c/src/sysdep.c` code.
